packages/block-library/src/navigation/placeholder-preview.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Icon, search } from '@wordpress/icons';
+
+const PlaceholderPreview = ( { hideSelected } ) => {
+	const classes = classNames( 'wp-block-navigation-placeholder__preview', {
+		'wp-block-navigation-placeholder__preview_select_hide': hideSelected,
```

Comment:
This doesn't appear to work for me. I can't get this class to appear in the editor.